libc/src/__support/UInt.h

```diff
@@ -579,19 +586,33 @@ struct BigInt {
     return *this;
   }
 
-  LIBC_INLINE constexpr uint64_t clz() {
-    uint64_t leading_zeroes = 0;
-    for (size_t i = WORD_COUNT; i > 0; --i) {
-      if (val[i - 1] == 0) {
-        leading_zeroes += WORD_SIZE;
-      } else {
-        leading_zeroes += countl_zero(val[i - 1]);
+  // TODO: remove and use cpp::countl_zero below.
```

Comment:
What was the issue encountered with using them?

Reply:
It's mostly about having a single way to do something.
Right now we have `big_int.clz()` and `cpp::countl_zero(big_int)`.